changelog.md

```diff
@@ -1,5 +1,16 @@
 # `bytemuck` changelog
 
+## Unreleased
+
+* Implement `Pod` and `Zeroable` for `__m512`, `__m512d` and `__m512i` without nightly.
+  Requires Rust 1.70, and is gated through the `avx512_simd` cargo feature.
```

Comment:
```suggestion
* Implement `Pod` and `Zeroable` for `core::arch::{x86, x86_64}::__m512`, `__m512d` and `__m512i` without nightly.
  Requires Rust 1.72, and is gated through the `avx512_simd` cargo feature.
```

Reply:
I don't know that we need the prefix part, but the correctly listed msrv is essential